src/bench/util/wallet_common.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php.
+
+#include <bench/util/wallet_common.h>
+
+#include <validationinterface.h>
+
+std::shared_ptr<CWallet> BenchLoadWallet(std::unique_ptr<WalletDatabase> database, WalletContext& context, uint64_t create_flags)
+{
+    bilingual_str error;
+    std::vector<bilingual_str> warnings;
+    auto wallet = CWallet::Create(context, "", std::move(database), create_flags, error, warnings);
+    NotifyWalletLoaded(context, wallet);
+    if (context.chain) {
+        wallet->postInitProcess();
+    }
```

Comment:
little topic:
it seems odd to re-use this two calls in other bench contexts.

`NotifyWalletLoaded` is looping over the wallet load callbacks, and `postInitProcess` syncs the wallet with the mempool, so guess that both will be no-op in most cases?.

Reply:
Yes, I think they'll be no-op for the most part.